code/modules/projectiles/projectile/beams.dm

```diff
@@ -54,6 +54,14 @@
 	damage = 40
 	armor_penetration = 20
 
+/obj/item/projectile/beam/midlaser/hegemony
+	damage = 40
+	armor_penetration = 20
```

Comment:
the description on the rifle says "A special crystal used in its design allows it to penetrate armor with pinpoint accuracy." but it actually has less AP and more damage than the normal laser rifle? 

moreover this has heavy laser cannon damage while having the same firerate/shots as the normal laser rifle, I'm not sure I agree with that

Reply:
I have never coded a gun before, and have no idea what is balanced or not. I'll tweak it appropriately.

Reply:
Gun has been tweaked to have lower damage and higher AP